lib/wanda/execution/evaluation.ex

```diff
@@ -224,4 +237,51 @@ defmodule Wanda.Execution.Evaluation do
   defp result_weight(:critical), do: 2
   defp result_weight(:warning), do: 1
   defp result_weight(:passing), do: 0
+
+  defp compute_values(values, facts, env) do
+    Enum.map(values, &value_from_conditions(&1, facts, env))
+  end
+
+  defp value_from_conditions(
+         %Wanda.Catalog.Value{
+           name: name,
+           default: default,
+           conditions: conditions
+         },
+         facts,
+         env
+       ) do
+    scoped_facts = map_facts_to_evaluation_scope(facts)
+
+    computed_value =
+      Enum.find_value(conditions, default, fn %Wanda.Catalog.Condition{
```

Comment:
Alias `Condition` at the top. This way, you can put the `Condition` declaration in one line even, which looks easier to